.github/workflows/people.yml

```diff
@@ -34,11 +34,14 @@ jobs:
 
   check-changelog:
     runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'noChangeLog') == false
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+        with:
+            fetch-depth: 0
       - name: Check that the CHANGELOG has been modified in the current branch
-        run: git whatchanged --name-only --pretty="" origin..HEAD | grep CHANGELOG
+        run: git whatchanged --name-only --pretty="" origin/main..HEAD | grep CHANGELOG
```

Comment:
Maybe do like it the `lint-git` job in case we cant to merge on another branch... 
```
origin/${{ github.event.pull_request.base.ref }}
```